packages/core/admin/admin/src/core/apis/content-manager.ts

```diff
@@ -36,7 +38,7 @@ interface Context {
   /**
    * The current content-type's model.
    */
-  model: string;
+  model: Common.UID.ContentType;
```

Comment:
you might find this harder to work with because you typically have to coerce strings to match it and handle if they don't, is there a particular reason you felt the need to be so narrow?

Reply:
ohk, type check fails for a request where contentType's type is not matching. Is there any other solution except casting?

Reply:
Where's the function that expects i? You can keep it, I just worry that it might cause more issues down the line.

Reply:
https://github.com/strapi/strapi/pull/19891/commits/2f626c2e1ef3d9734e5f250adb8cba5a2bafefe3#diff-05bcca23781be51738912dc97977b65539ab304a0908604f9de8445d98306a6cR73